common/url_constants.cc

```diff
@@ -15,4 +15,3 @@ const char kWidevineTOS[] = "https://policies.google.com/terms";
 const char kRewardsUpholdSupport[] = "https://uphold.com/en/brave/support";
 const char kP3ALearnMoreURL[] = "https://brave.com/P3A";
 const char kP3ASettingsLink[] = "chrome://settings/privacy";
-const char kWaybackQueryURL[] = "https://archive.org/wayback/available?url=";
```

Comment:
Deleted because this constant is deprecated one.
Same constant in components/brave_wayback_machine/wayback_machine_url_fetcher.cc was used.
I think I forgot to delete this after refactoring that moves wayback implementation to component layer.
Create new `url_constatns.[h|cc]` in `components/brave_wayback_machine/` for this constant.